packages/common/src/core/schemas/internal/getLayoutSetupUiSchemaElement.ts

```diff
@@ -0,0 +1,38 @@
+import { IUiSchemaElement } from "../types";
+
+/**
+ * When creating an entity, an editor can elect to choose
+ * a layout for the new entity. The following util builds
+ * the UiSchema elements to render this configuration in
+ * various creation flows.
+ *
+ * @param i18nScope intl - scope for translations
+ * @param context contextual - portal & auth information
+ */
+export function getLayoutSetupUiSchemaElement(
+  i18nScope: string
+): IUiSchemaElement[] {
+  const siteOrPageString = i18nScope.includes("site")
+    ? "{{shared.fields._layoutSetup.type.siteLayout:translate}}"
```

Comment:
not a huge deal, but I'd suggest moving the label translations out of the "shared" namespace and into their type-specific namespaces to make this logic more generic for when we hook this up for other entities:

```ts
const label = `{{${i18nScope}.fields._layoutSetup.label}}`
...
return [
  {
    label,
    ...
  }
]
```